src/SiteHeader.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import MediaQuery from 'react-responsive';
+import Responsive from 'react-responsive';
```

Comment:
Changing this to `Responsive` was not strictly necessary, I don't think, but I'm sorta down with the attribute signature of it as opposed to `MediaQuery`'s query string syntax.  So I left it.  I originally changed it because I thought it might be part of my problem getting the media queries to work.